l10n_br_stock_account/models/stock_picking.py

```diff
@@ -121,3 +103,43 @@ def _generate_document_number(self):
             )
             self.document_serie = self.document_serie_id.code
             self.document_number = self.document_serie_id.next_seq_number()
+
+    def _get_default_fiscal_operation(self):
+        """
+        Metodo usado para ser herdado pelos modulos
+        l10n_br_sale_stock
+        l10n_br_purchase_stock
+        """
+        company = self.env.company
```

Comment:
melhor em inglês, algo como; "Meant to be overriden in modules such as l10n_br_sale, l10n_br_purchase_stock..."